src/common/adapters/notion.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * notion.so adapter
+ *
+ * The adapter extracts the UUID of a selected notion.so task ("page") from the
+ * page URL and uses the notion.so API to retrieve the corresponding task
+ * information.
+ *
+ * Note: this adapter uses notion.so's internal v3 API as there is no public
+ * API yet.
+ */
```

Comment:
Neat documentation. 👌

I noticed I usually just use `//` for commenting, even when the comment consists of multiple lines. 🙈 I found this convenient, when your (previously) one-line comment becomes longer and suddenly spans multiple lines.

However, I also checked the Airbnb [styleguide](https://github.com/airbnb/javascript#comments) which we're using as a basis and they say to use `/** … */`for multi-line comments.

Do you have any preference/opinion? Should we update those other multi-line comments (in a separate PR)? 🤔

Reply:
I find this closer to how documentation looks like in Elixir and felt like it's a better fit for these documentation snippets on top of files. I discovered it in the wordpress and [google](https://google.github.io/styleguide/jsguide.html#jsdoc-top-file-level-comments) styleguides as well.

No strong feelings, but if you think it's visually cleaner, I can also update the rest of the class comments. Would also be a good opportunity to check whether they're still up to date 😉